### ISSUE TITLE
Toolbar documentation is wrong

### ISSUE THREAD

**Issue by loranallensmith** [loranallensmith/brightwheel#3]
The example in the [current documentation for toolbar](https://github.com/loranallensmith/brightwheel/blob/50a9b5964825fa4455745ed9caad10ff790fd544/docs/README.md#toolbarproperties-children) appears to be incorrect.

It lists properties in an `attributes` object when they should be outside.

```javascript
let myBar = new Toolbar({
  attributes: {
    id: 'bar-1',
    classNames: ['my-class'],
    title: 'My Toolbar',
    type: 'header'
  }
}, [child-component, child-component]);
```

I believe it should be:

```javascript
let myBar = new Toolbar({
  attributes: {
    id: 'bar-1',
  },
  classNames: ['my-class'],
  title: 'My Toolbar',
  type: 'header'
}, [child-component, child-component]);
```

For reference, the code for this feature is in [`src/toolbar.js`](https://github.com/loranallensmith/brightwheel/blob/master/src/toolbar.js#L16).

**Comment by loranallensmith** [comment]
This was fixed in #4.  Thanks again @jose4125!  ✨  :balloon: :tada:

**Comment by aviaryan** [comment]
If it is fixed, please close this issue. Thanks :beers:
